Formula/p/podlet.rb

```diff
@@ -0,0 +1,27 @@
+class Podlet < Formula
+  desc "Generate podman quadlet files from a podman command or compose file"
+  homepage "https://github.com/containers/podlet"
+  url "https://github.com/containers/podlet/archive/refs/tags/v0.2.4.tar.gz"
```

Comment:
should be 0.3.0? can you raise an issue to ask about it?

- https://github.com/containers/podlet/pull/77